src/NServiceBus.Transport.Msmq/Persistence/SubscriptionStorage/MsmqSubscriptionPersistence.cs

```diff
@@ -20,11 +20,11 @@ protected override void Setup(FeatureConfigurationContext context)
                     // The user has not configured the subscriptions queue to be "NServiceBus.Subscriptions" but there's a local queue. 
                     // Indicates that the endoint was using old default queue name.
                     throw new Exception(
-                        "Detected the presence of an old default queue named `NServiceBus.Subscriptions`. The new default is now `[Your endpointname].Subscriptions`. Move the messages to the new queue or configure the subscriptions queue name.");
+                        "Detected the presence of an old default queue named `NServiceBus.Subscriptions`. The new default is now `[Your endpointname].Subscriptions`. Move the relevant subscription messages to the new queue or configure the subscriptions queue name.");
```

Comment:
Should we add "see our documentation for instructions how to migrate subscription data"?

Reply:
@andreasohlund - Yeah. Since we'll be also be adding detailed instructions on how to do this in our docs.

Reply:
@andreasohlund how about: 

Detected the presence of an old default queue named `NServiceBus.Subscriptions`. Either migrate the subscriptions to the new default queue `[Your endpointname].Subscriptions`, see our documentation for more details, or explicitly configure the subscriptions queue name.

Reply:
>or explicitly configure the subscriptions queue name

or explicitly configure the subscriptions queue name to `NServiceBus.Subscriptions` if you want to use the existing queue.

?

Reply:
@andreasohlund - yep. that's perfect.